Mathlib/RingTheory/MvPowerSeries/LinearTopology.lean

```diff
@@ -42,7 +43,7 @@ namespace LinearTopology
 
 open scoped Topology
 
-open Set SetLike
+open Set SetLike Filter
```

Comment:
This suggests that the linter / the mathlib writing recommendations could insist that some namespaces are opened. Either because it is standard, or because the namespace had to be provided at too many places.